src/defaultfont.cpp

```diff
@@ -77,14 +77,14 @@ namespace gcn
         return 8*text.size();
     }
 
-    int DefaultFont::drawGlyph(Graphics* graphics, unsigned char glyph, int x, int y)
+    int DefaultFont::drawGlyph(Graphics* graphics, unsigned char glyph, const int x, const int y)
```

Comment:
It is even more discutable when it is not consistant:
```
int DefaultFont::drawGlyph(Graphics*const graphics, const unsigned char glyph, const int x, const int y)
```
That's why I'm not sure for parameter.